build.gradle.kts

```diff
@@ -1,9 +1,4 @@
 plugins {
-    id("com.automattic.android.publish-to-s3")
-}
-
-s3PublishPlugin {
-    groupId = "com.automattic.android"
-    artifactId = "publish-to-s3"
+    id("com.automattic.android.publish-to-s3-helpers")
```

Comment:
Gradle noob question:

Is this defining a dependency on the helper plugin, which is part fo the project? If so why is it needed?

I tried to look in the docs, e.g. [this](https://docs.gradle.org/current/userguide/kotlin_dsl.html), but I can find an explanation for this syntax.

Thanks!

Reply:
Yes, this applies the helpers plugin. It's not needed, we just use it to be able to test the integration. (functional tests already cover this, but it's easier to use this for debugging)

This syntax is called Plugin DSL and you can find its documentation here: https://docs.gradle.org/current/userguide/plugins.html - Make sure to select `Kotlin` for the examples since this is a kotlin script and not a groovy one.